app/models/item.rb

```diff
@@ -6,6 +6,8 @@ class Item < ApplicationRecord
   has_many :orders_items
   has_many :orders, through: :orders_items
 
+  enum status: %w(active retired)
```

Comment:
Cool. I think these are all we need.

Reply:
My favorite thing about enums is how they give you options. Life is all about options.